pyccl/tests/test_cclobject.py

```diff
@@ -0,0 +1,189 @@
+import pytest
+import numpy as np
+import pyccl as ccl
+import functools
+
+
```

Comment:
Can you test that disabling fancy_repr does the right thing?

Reply:
Added in the tests.